src/main/java/mate.academy/model/Product.java

```diff
@@ -9,6 +9,9 @@ public class Product {
     private String description;
     private BigDecimal price;
 
+    public Product(String name, double price) {
```

Comment:
The 'price' parameter should be of type 'BigDecimal' to match the 'price' field type of the 'Product' class. Using 'double' can lead to precision issues with monetary values.